test/e2e/operator_groups_e2e_test.go

```diff
@@ -105,39 +103,33 @@ func TestOperatorGroup(t *testing.T) {
 
 	oldCommand := patchOlmDeployment(t, c, otherNamespaceName)
 
-	log.Debug("Creating CSV")
```

Comment:
I can understand removing the default log level of debug, but for a five minute long test I'd rather have some real time output for the logging.

Reply:
```
# --- PASS: TestOperatorGroup (11.61s)
```
looks like it's running in 12s?

Reply:
After the explanation of above, this is fine (that is logging with t.Log).